scripts/conversion_script.py

```diff
@@ -57,7 +57,7 @@ def __init__(
         self.error = error  # true if the script wasn't able to finish, otherwise false
         self.message = message
         self.report = report
-        self.tasks_format_version = "1.0"
+        self.tasks_format_version = "1.1"
```

Comment:
I don't think this needs a version bump, as nothing changed in the format

Reply:
I'll revert this change